core/modules/layout/css/layout.flexible.admin.css

```diff
@@ -0,0 +1,82 @@
+/**
+ * Flexible template admin css.
+ */
+
+ .layout-flexible-region-top .label {
+   display: inline-block;
+   margin: 0;
+   padding: 4px 12px 4px 0; /* LTR */
```

Comment:
There's no matching `[dir="rtl"]` for this

Reply:
added